ui/app/adapters/clients/activity.js

```diff
@@ -39,8 +39,15 @@ export default class ActivityAdapter extends ApplicationAdapter {
 
   queryRecord(store, type, query) {
     const url = `${this.buildURL()}/internal/counters/activity`;
-    const queryParams = this.formatQueryParams(query);
-    return this.ajax(url, 'GET', { data: queryParams }).then((resp) => {
+    const options = {
+      data: this.formatQueryParams(query),
```

Comment:
Since we don't need to format the namespace I wonder if this should be renamed to `formatTimeQuery` and the other function could just be simplified to `formatTimeStamp`. Just a thought, but definitely not necessary!

Reply:
Going to keep it for now since I have some other changes in flight, but might rename on a next iteration!